packages/clerk-js/src/ui/components/SubscriptionDetails/index.tsx

```
@@ -35,10 +35,12 @@
   Flex,
   Heading,
   localizationKeys,
+  Span,
```

ESLint

[error] 'Span' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)